src/components/product-card/product-card.component.html

```diff
@@ -0,0 +1,50 @@
+<section> <!-- I have a section tag here, however, I feel that it may need to be removed if we dynamically add multiple of these components-->
+    <div class="container py-5">
+      <div>
+        <div class="col-md-8 col-lg-6 col-xl-4">
+          <div class="card">
+            <img src="/assets/spongebobChair.jpg"
+              class="card-img-top" alt="spongebob chair" />
+            <div class="card-body">
+              <div class="text-center">
+                <h5 class="card-title">ITEM NAME</h5>
+                <p class="text-muted mb-4">BRIEF ITEM DESCRIPTION</p>
+              </div>
+              <div>
+                <div class="d-flex justify-content-between">
+                  <span>PRICING DETAILS</span><span>$5,999</span>
```

Comment:
All these should also not be hard coded, but it's fine for now until we figure out what we want the cards to look like.